schema/src/generated/types.ts

```diff
@@ -63,7 +63,7 @@ export type Mutation = {
   login: AuthPayload;
   addNewbieTask: Task;
   addBuddyTask: Task;
-  deleteTask: Task;
+  deleteTask: Newbie;
```

Comment:
Why `deletedTask` is now a `Newbie` type?

Reply:
We need `Newbie` to auto update apollo cache